scrapers/MindGeekAPI.py

```diff
@@ -422,10 +414,10 @@ def scraping_json(api_json, url=""):
     except:
         url_sceneid = None
     if url_sceneid is None:
-        debug("[ERROR] Can't get the ID of the Scene. Are you sure that URL is from Mindgeek Network?")
+        log.error("Can't get the ID of the Scene. Are you sure that URL is from Mindgeek Network?")
         sys.exit()
     else:
-        debug("[INFO] ID: {}".format(url_sceneid))
+        log.info("ID: {}".format(url_sceneid))
```

Comment:
```suggestion
        log.debug("ID: {}".format(url_sceneid))
```
Only useful for debugging